README.md

```diff
@@ -25,15 +25,29 @@ This project uses a [lexer/parser](http://github.com/graphql-dotnet/parser) orig
 > If errors occur, it is recommended that you first check the behavior on the latest available preview version before
 > reporting a issue.
 
-You can install the latest version via [NuGet](https://www.nuget.org/packages/GraphQL/).
+You can install the latest version via NuGet.
 
+If you're targeting .NET Core 3+, you should use [GraphQL.SystemTextJson](https://www.nuget.org/packages/GraphQL.SystemTextJson/).
```

Comment:
```suggestion
If you're targeting .NET Core 3+, you are advised to use [GraphQL.SystemTextJson](https://www.nuget.org/packages/GraphQL.SystemTextJson/).
```

Reply:
I'd say there's no point even suggesting otherwise, and it's highly risky like the guys from MS have said. I'd prefer to not even suggest it.

Reply:
As you wish